### PR TITLE
KEP: Windows GMSA support

### PR DESCRIPTION
**Pull request by PatrickLang**
Starting a KEP to supercede https://github.com/kubernetes/kubernetes/issues/62038

I'd like to develop this with wg-container-identity, sig-auth, and sig-windows, and it seems like KEP is the easiest way to capture feedback and version control the proposal until the code is implemented.

Will also resolve key ask from https://github.com/kubernetes/kubernetes/issues/51691

cc @rpsqrd

### CODE REVIEW

File: keps/sig-auth/draft-20180620-windows-group-managed-service-accounts.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
+---
+kep-number: 0
+title: Windows Group Managed Service Accounts for Container Identity
+authors:
+  - "@patricklang"
+owning-sig: "wg-container-identity"
```
Comment (mattfarina) [inline keps/sig-auth/draft-20180620-windows-group-managed-service-accounts.md]: The docs note that a SIG needs to own this. See https://github.com/kubernetes/community/blob/master/keps/0001-kubernetes-enhancement-proposal-process.md#kep-metadata

Would sig-auth or sig-windows be a good owner?

File: keps/sig-auth/draft-20180620-windows-group-managed-service-accounts.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+
+
+## Proposal
+
+### Background
+
+#### What is Active Directory?
+Windows applications and services typically use Active Directory identities to facilitate authentication and authorization between resources. In a traditional virtual machine scenario, the computer is joined to an Active Directory domain which enables it to use Kerberos, NTLM, and LDAP to identify and query for information about other resources in the domain. When a computer is joined to Active Directory, it is given an unique identity represented as a computer object in LDAP.
+
+#### What is a Windows service account?
```
Comment (mattfarina) [inline keps/sig-auth/draft-20180620-windows-group-managed-service-accounts.md]: Are these service accounts or service principals?

**Comment by justaugustus** [comment]
REMINDER: KEPs are moving to [k/enhancements](https://github.com/kubernetes/enhancements) on November 30. Please attempt to merge this KEP before then to signal consensus.
For more details on this change, review [this thread](https://groups.google.com/d/msg/kubernetes-dev/LIkZoIqCT20/t515CV2mAgAJ).

Any questions regarding this move should be directed to that thread and not asked on GitHub.

**Comment by justaugustus** [comment]
KEPs [have moved](https://github.com/kubernetes/enhancements/pull/638) to [k/enhancements](https://github.com/kubernetes/enhancements). 
This PR will be closed and any additional changes to this KEP should be submitted to k/enhancements.
For more details on this change, review [this thread](https://groups.google.com/d/msg/kubernetes-dev/LIkZoIqCT20/t515CV2mAgAJ).

Any questions regarding this move should be directed to that thread and not asked on GitHub.
/close

**Pull request closed without merging**